paparazzi/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionTest.kt

```diff
@@ -125,6 +125,7 @@ class AccessibilityRenderExtensionTest {
           val expected = File("src/test/resources/${snapshot.name}.png")
           ImageUtils.assertImageSimilar(
             relativePath = expected.path,
+            failureDir = expected.parentFile,
```

Comment:
similar to previous comment:

if the test fails, will this directory even be writable? if so, will it be cleaned up?  let's pass a temporary folder instead?